File: src/main/java/no/nav/personopplysninger/features/endreopplysninger/domain/kontonummer/Bank.java
package no.nav.personopplysninger.features.endreopplysninger.domain.kontonummer;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Bank {
    private String adresseLinje1;
    private String adresseLinje2;
    private String adresseLinje3;
    private String kode;
    private String navn;

    public String getAdresseLinje1() {
        return adresseLinje1;
    }

    public String getAdresseLinje2() {
        return adresseLinje2;
    }

    public String getAdresseLinje3() {
        return adresseLinje3;
    }

    public String getKode() {
        return kode;
    }

    public String getNavn() {
        return navn;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/endreopplysningergammel/domain/kontonummer/Bank.kt
package no.nav.personopplysninger.features.endreopplysningergammel.domain.kontonummer

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class Bank {
    val adresseLinje1: String? = null
    val adresseLinje2: String? = null
    val adresseLinje3: String? = null
    val kode: String? = null
    val navn: String? = null
}
